recipes/r-fillpattern/meta.yaml

```
@@ -0,0 +1,78 @@
+{% set version = '1.0.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-fillpattern
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/fillpattern_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/fillpattern/fillpattern_{{ version }}.tar.gz
+  sha256: 1c5c0a4a2651dfbeb919a619c3bebd8ac3ec3186f0607d7f7a8aeeefc7cada77
+
+build:
+  skip: True  # [win]
```

This is a `noarch` package. It is acceptable for the Windows build to fail - once merged, the feedstock will only build on `linux-64` platform.

```suggestion
  noarch: generic
```